lte/gateway/c/core/oai/test/lib/mme_app/test_mme_app_map.cpp

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Thanks for the review and providing the comments
In previous commits, test_mme_app_map.cpp was part of test/mme_app_task. But @ssanadhya was in opinion to have as part of common library. Because here we are validating the operation on map like, insert, remove etc.
Within lib, we created mme_app folder because on each task, we can execute stability tests